JT400 flaky tests with real srvice

### Bug description

Several flaky failures were spotted:

```
org.awaitility.core.ConditionTimeoutException: Lambda expression in org.apache.camel.quarkus.component.jt400.it.Jt400Test expected "true" but was null within 30 seconds.
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.awaitility.core.AbstractHamcrestCondition.await(AbstractHamcrestCondition.java:86)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1160)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:712)
	at org.apache.camel.quarkus.component.jt400.it.Jt400Test.testInquiryMessageQueue(Jt400Test.java:259)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.test.junit.QuarkusTestExtension.runExtensionMethod(QuarkusTestExtension.java:1000)
	at io.quarkus.test.junit.QuarkusTestExtension.interceptTestMethod(QuarkusTestExtension.java:848)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:101)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:81)
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:103)
	... 9 more
```

--- a/integration-tests/jt400/src/main/java/org/apache/camel/quarkus/component/jt400/it/Jt400Resource.java
+++ b/integration-tests/jt400/src/main/java/org/apache/camel/quarkus/component/jt400/it/Jt400Resource.java
@@ -37,13 +37,17 @@ import org.apache.camel.CamelContext;
 import org.apache.camel.ConsumerTemplate;
 import org.apache.camel.Exchange;
 import org.apache.camel.ProducerTemplate;
+import org.apache.camel.ServiceStatus;
 import org.apache.camel.component.jt400.Jt400Component;
 import org.apache.camel.component.jt400.Jt400Endpoint;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Path("/jt400")
 @ApplicationScoped
 public class Jt400Resource {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Jt400Resource.class);
 
     @ConfigProperty(name = "cq.jt400.url")
     String jt400Url;
@@ -165,12 +169,17 @@ public class Jt400Resource {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     public Response stopRoute(@PathParam("route") String routeName) throws Exception {
+        LOGGER.info("Stopping route: {}", routeName);
+        ServiceStatus routeStatus = context.getRouteController().getRouteStatus(routeName);
         if (context.getRouteController().getRouteStatus(routeName).isStoppable()) {
+            LOGGER.info("Route {} before stop information: {}", routeName, routeStatus.toString());
             context.getRouteController().stopRoute(routeName);
+            LOGGER.info("Route {} stopped", routeName);
         }
-        boolean resp = context.getRouteController().getRouteStatus(routeName).isStopped();
+        routeStatus = context.getRouteController().getRouteStatus(routeName);
+        LOGGER.info("Route {} status: {}", routeName, routeStatus.toString());
 
-        return Response.ok().entity(resp).build();
+        return Response.ok().entity(routeStatus.isStopped()).build();
     }
 
     @Path("/component/stopWrong")

--- a/integration-tests/jt400/src/test/java/org/apache/camel/quarkus/component/jt400/it/Jt400Test.java
+++ b/integration-tests/jt400/src/test/java/org/apache/camel/quarkus/component/jt400/it/Jt400Test.java
@@ -256,7 +256,7 @@ public class Jt400Test {
         LOGGER.debug("testInquiryMessageQueue: inquiry message processed");
 
         //stop route (and wait for stop)
-        Awaitility.await().atMost(WAIT_IN_SECONDS, TimeUnit.SECONDS).until(
+        Awaitility.await().atMost(2 * WAIT_IN_SECONDS, TimeUnit.SECONDS).until(
                 () -> RestAssured.get("/jt400/route/stop/inquiryRoute")
                         .then()
                         .statusCode(200)
@@ -265,7 +265,7 @@ public class Jt400Test {
         LOGGER.debug("testInquiryMessageQueue: inquiry route stooped");
 
         //check written message with client
-        Awaitility.await().pollInterval(1, TimeUnit.SECONDS).atMost(20, TimeUnit.SECONDS).until(
+        Awaitility.await().pollInterval(1, TimeUnit.SECONDS).atMost(WAIT_IN_SECONDS, TimeUnit.SECONDS).until(
                 () -> getClientHelper().peekReplyToQueueMessage(replyMsg),
                 Matchers.notNullValue());
         LOGGER.debug("testInquiryMessageQueue: reply message confirmed by peek: " + replyMsg);

